robot_smach_states/src/robot_smach_states/navigation/find.py

```diff
@@ -105,11 +106,17 @@ def __init__(self, robot, knowledge, source_entity_designator, description_desig
                                    Inspect(robot=robot,
                                            entityDes=source_entity_designator,
                                            objectIDsDes=segmented_entities_designator,
-                                           searchArea=area_name_designator,
+                                           searchArea=area_name,
                                            navigation_area=navigation_area_designator),
-                                   transitions={'done': 'CHECK_IF_ENTITY_FOUND',
+                                   transitions={'done': 'WAIT',
```

Comment:
Please explain why we need to wait for a second, not only in this PR, but in the code to please.

Reply:
I would still like to know why we have a seemingly random wait of half a second. If I see it like this I don't know what it is for and I would throw it out

Reply:
We need to wait for some time in order to give ed time to process the updaterequest. I didn't time this exactly and it is likely not constant so it might be 0.1 second or 0.2 or 0.5.

Reply:
Shouldn't this be handled in the inspect itself? I would agree with adding a sleep there, but not here.

Reply:
I'll create an issue for this and revert these changes as this goes outside of the scope of this PR